community/mplayer/APKBUILD

```diff
@@ -4,22 +4,34 @@ pkgver=1.4.0
 pkgrel=1
 pkgdesc="A movie player for linux"
 url="http://www.mplayerhq.hu/design7/news.html"
-arch="x86_64"
+arch="x86_64 aarch64"
 license="GPL-2.0-or-later"
 subpackages="$pkgname-doc"
 makedepends="libxxf86dga-dev libxv-dev libmad-dev lame-dev libao-dev
-	libtheora-dev xvidcore-dev zlib-dev sdl-dev freetype-dev
-	x264-dev faac-dev ttf-dejavu libxvmc-dev alsa-lib-dev live-media-dev
-	mesa-dev yasm libpng-dev libvdpau-dev libvpx-dev libcdio-paranoia-dev"
+	libtheora-dev xvidcore-dev zlib-dev sdl-dev freetype-dev faac-dev
+	x264-dev x265-dev ttf-dejavu libxvmc-dev alsa-lib-dev live-media-dev
+	mesa-dev yasm libpng-dev libvdpau-dev libvpx-dev libcdio-paranoia-dev
+	gnutls-dev libvorbis-dev imlib2-dev coreutils bzip2-dev perl-dev
+	sdl2-dev libxfixes-dev libva-dev v4l-utils-dev opus-dev libass-dev"
 
 source="http://www.mplayerhq.hu/MPlayer/releases/MPlayer-${pkgver%.*}.tar.xz
 	x11-libs.patch
+	conf.patch
 	"
 builddir="$srcdir"/MPlayer-${pkgver%.*}
 build() {
+	case "$CARCH" in
+		aarch64)
+			cpudetection="disable-runtime-cpudetection"
+			vdpau="disable-vdpau"	;;
+		*)
+			cpudetection="enable-runtime-cpudetection"
+	esac
+
 	./configure \
 		--prefix=/usr \
-		--enable-runtime-cpudetection \
+		--$cpudetection	\
+		--$vdpau	\
```

Comment:
This doesn't work, if you don't set `vdpau` you get `unknown argument: --` as error.